lib/puppet/provider/firewall/ip6tables.rb

```diff
@@ -45,7 +45,8 @@
 
 
   def initialize(*args)
-    if Facter.fact('ip6tables_version').value.match /1\.3\.\d/
+    ip6tables_version = Facter.fact('ip6tables_version').value
```

Comment:
This can be replaced with:
```ruby
ip6tables_version = Facter.value('ip6tables_version')
```
Or, possibly superior:
```ruby
ip6tables_version = Facter.value(:ip6tables_version)
```